### PR TITLE
move subs to outer function

### PR DESCRIPTION
**Pull request by sid597**

### CODE REVIEW

File: src/cljs/athens/listeners.cljs [hunk 1]
```diff
@@ -266,35 +266,37 @@
...
-                            "Press Cancel to wait, or OK to leave without saving (will cause data loss!).")
-                       (fn []
-                         (reset! force-leave true)
-                         (js/window.close))
-                       #()])
-            (.. e preventDefault)
-            (set! (.. e -returnValue) "Setting e.returnValue to string prevents exit for some browsers.")
-            "Returning a string also prevents exit on other browsers.")
+  (let [synced (rf/subscribe [:db/synced])
+        selected-db (rf/subscribe [:db-picker/selected-db])]
```
Comment (filipesilva) [inline src/cljs/athens/listeners.cljs:270]: This change is not necessary, the function is not part of a regent component.

**Review by filipesilva — APPROVED** [review]
LGTM, has conflicts though.

**Comment by sid597** [comment]
We will do this as we go insted of doing in one shot because this will cause merge conflicts with many branches

**Comment by tangjeff0** [comment]
Great idea!

**Pull request closed without merging**